build-live.sh

```diff
@@ -0,0 +1,304 @@
+#!/usr/bin/env bash
+
+set -euio pipefail
+set -x
+
+export WORK=~/work
+export CD=~/cd
+export FORMAT=squashfs
+export FS_DIR=casper
+export ARCH=amd64
+export DIST=trusty
+export DEBIAN_FRONTEND=noninteractive
+export DEBCONF_NONINTERACTIVE_SEEN=true
+export CONTAINER=live
+
+# TODO start with an ubuntu-14.04-desktop ISO extracted into ${CD}
+mkdir -p ${CD}/{${FS_DIR},boot/grub,preseed}
+
+sudo apt update
+sudo apt install -y grub2 xorriso squashfs-tools debootstrap
+
+cd $WORK
+lxc launch ubuntu:${DIST}/${ARCH} $CONTAINER -p default -p docker \
+  -c environment.DEBCONF_NONINTERACTIVE_SEEN=true \
+  -c environment.DEBIAN_FRONTEND=noninteractive \
+  -c security.privileged=true
+
+# wait for the system to start
+sleep 5
+
+lxc exec $CONTAINER -- apt update
+lxc exec $CONTAINER -- apt upgrade -y
+lxc exec $CONTAINER -- apt install -y ubuntu-standard
+lxc exec $CONTAINER -- apt purge -y cloud-init
+
+# install POSM
+
+lxc exec $CONTAINER -- apt install -y --no-install-recommends git
+lxc exec $CONTAINER -- git clone https://github.com/americanredcross/posm-build
+set +e
+# TODO split wifi into package installation and configuration
+lxc exec $CONTAINER -- /root/posm-build/kickstart/scripts/bootstrap.sh base nodejs ruby gis mysql postgis nginx osm fieldpapers omk tl carto tessera admin samba blink1 docker
+set -e
+
+lxc exec $CONTAINER -- apt install --no-install-recommends -y linux-image-generic-lts-xenial wireless-tools
+lxc exec $CONTAINER -- apt install --no-install-recommends -y \
+  dnsmasq \
+  dnsmasq-utils \
+  hostapd \
+  iw \
+  rfkill \
+  rng-tools
+
+# grab the list of packages
+lxc exec $CONTAINER -- dpkg-query -W --showformat='${Package} ${Version}\n' | sort > ${CD}/${FS_DIR}/filesystem-desktop.manifest
+
+lxc exec $CONTAINER -- apt install -y --no-install-recommends casper
+
+echo "Diverting running-in-container..."
```

Comment:
Add a comment to explain why this is being diverted.

(It's being diverted because it returns false when `initctl` is also diverted, as the first check uses `upstart` and fails because the service being queried intentionally wasn't started.)